### PR TITLE
Add space for MANTRA DAO

### PR DESCRIPTION
**Pull request by spaceforce-dev**
**Add space for MANTRA DAO**  
  
Etherscan: https://etherscan.io/address/0x2baecdf43734f22fd5c152db08e3c27233f0c7d2
CoinGecko: https://www.coingecko.com/en/coins/mantra-dao
Website: https://www.mantradao.com
Twitter: https://twitter.com/mantradao

### CODE REVIEW

File: spaces/mantra-dao/index.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "token": "0x2baEcDf43734F22FD5c152DB08E3C27233F0c7d2",
```
Comment (gokaiorg) [inline spaces/mantra-dao/index.json:2]: You will have to remove the skin field. We do not use it anymore. Please refer to the documentation https://docs.snapshot.page/guides/create-a-space#3-space-metadata

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The images file size should not exceed 50KB. Please reduce their weight.

**Comment by gokaiorg** [comment]
Related to https://github.com/snapshot-labs/snapshot.js/pull/29

**Comment by spaceforce-dev** [comment]
Thanks @gokaiorg 

I cant find where it mentions skins aren't used anymore, can you help point me in the right direction?'

**Comment by spaceforce-dev** [comment]
Or do i just simply remove skin the line from my index.js

**Comment by gokaiorg** [comment]
@spaceforce-dev I was talking about the field token sorry!

**Comment by spaceforce-dev** [comment]
Should have all required updates now 👍

**Review by gokaiorg — CHANGES_REQUESTED** [review]
The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image
image

**Comment by spaceforce-dev** [comment]
OOps thanks for spotting that.

Has this change i just pushed made it readable? i want the text to be light ie white

**Comment by spaceforce-dev** [comment]
Screen Shot 2020-11-09 at 2 12 51 pm

**Comment by spaceforce-dev** [comment]
i'm trying to get it to look similar to this mockup.

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**